scripts/utils/migrate/html-to-markdown.js

```diff
@@ -193,6 +193,14 @@ module.exports = (file) => {
       filter: (node) => node.nodeName === 'I' && node.classList.contains('fa'),
       replacement: (content, node) =>
         htmlToJSXConverter.convert(node.outerHTML),
+    })
+    .addRule('var/mark', {
+      filter: ['var', 'mark'],
+      replacement: (content, node) => {
+        const tag = node.nodeName.toLowerCase();
+
+        return `<${tag}>${content}</${tag}>`;
```

Comment:
how is this different than just returning the node ( `<var>` or `<mark>` ))?

Reply:
In this case, I'm filtering this rule by both `var` and `mark`. The `replacement` function doesn't know which is which when it is called, so this just makes sure that it takes the `nodeName` and uses that as the tag. The alternative was this (which didn't feel any better)

```js
return node.nodeName.toLowerCase() === 'var' ? `<var>${content}</var>` : `<mark>${content}</mark>`
```